core/tests/webdriverio_utils/users.js

```diff
@@ -205,7 +205,7 @@ var createUserWithRole = async function (email, username, role) {
   // fails the test with error '$ is not defined for Admin.js file'.
   var adminPage = new AdminPage.AdminPage();
   await _createFirebaseAccount(email, true);
-  await login(email);
+  login(email);
```

Comment:
Why has `await` been removed?

Reply:
@StephenYu2018 Actually, this was intentional. I was unable to run e2e tests on local, so made a deliberate bug to confirm the solution.